<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GMT_psxy</class>
 <widget class="QDialog" name="GMT_psxy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;æŒ‰TABé”®å¯åœ¨å„é€‰é¡¹é—´å¿«é€Ÿåˆ‡æ¢&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="g_color">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>30</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-G å¡«å……è‰²</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="L_option">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>311</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>-L é—­åˆå¤šè¾¹å½¢ã€‚æœ¬é¡¹æœªå‹¾é€‰æ—¶ï¼Œ-Gå¯¹äºç”»çº¿æ®µæ— æ•ˆ</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="draw_symbol">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>87</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>ç”»ç¬¦å·</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="draw_line">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>ç”»çº¿æ®µæˆ–å¤šè¾¹å½¢</string>
   </property>
  </widget>
  <widget class="QComboBox" name="S_option">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>-St â–³ä¸‰è§’å½¢</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-Sa ğŸŸ‰äº”è§’æ˜Ÿ</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-Sc åœ†</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-Sx å‰å·</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-Sd â¬¦è±å½¢</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-Sg â¯ƒå…«è¾¹å½¢</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-Sh â¬£å…­è¾¹å½¢</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-Si â–½å€’ä¸‰è§’</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-Sn â¯‚äº”è¾¹å½¢</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-Ss â–¡æ­£æ–¹å½¢</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-S- çŸ­æ¨ªçº¿</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-S+ åŠ å·</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-Sy çŸ­ç«–çº¿</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="W_pen_color">
   <property name="geometry">
    <rect>
     <x>226</x>
     <y>30</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>çº¿é¢œè‰²</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>31</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-W</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="W_pen_w">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>30</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>å¡«å†™å®½åº¦</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>32</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#b5b5b5;&quot;&gt;ç¬¦å·è½®å»“æˆ–çº¿æ®µå®½åº¦ä»¥åŠé¢œè‰²&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="other_options">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>289</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#868686;&quot;&gt;å‰©ä½™æœªæ¶‰åŠé€‰é¡¹å¯è‡ªç”±æ·»åŠ ï¼š&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bok">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ç¡®å®š</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bexit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>å–æ¶ˆ</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="size">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>60</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>å¡«å†™å¤§å°</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>62</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#b5b5b5;&quot;&gt;-S ç¬¦å·å¤§å°(size)ç”¨å¤–æ¥åœ†ç›´å¾„è¡¨ç¤º&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>531</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>16</width>
     <height>581</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>113</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>431</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>é€‰æ‹©æ•°æ®è¾“å…¥çš„æ–¹å¼ï¼š</string>
   </property>
   <widget class="QRadioButton" name="use_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>87</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>ç›´æ¥è¾“å…¥</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="position">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS UI Gothic'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS UI Gothic';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string>è¯·åœ¨å³å›¾ç”¨é¼ æ ‡ç‚¹å‡»å†³å®šå•ç‚¹åæ ‡ä½ç½®ã€‚</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="position_input">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>è¯·åœ¨è¿™é‡Œè¾“å…¥ x y æ ¼å¼çš„å•ç‚¹åæ ‡ã€‚</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="use_mouse">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>é¼ æ ‡ç‚¹å‡»é€‰æ‹©</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="use_file">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>ä½¿ç”¨æ–‡ä»¶è¾“å…¥</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_datafile">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>é€‰æ‹©æ•°æ®æ–‡ä»¶</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="datafile">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>ä»…æ”¯æŒå¤šè¡Œæ–‡æœ¬æ–‡ä»¶ï¼Œæ¯è¡Œ x y ä¸ºæ¯ä¸ªç‚¹åæ ‡</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="use_G">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>31</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="use_W">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>30</y>
     <width>21</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>use_G</tabstop>
  <tabstop>g_color</tabstop>
  <tabstop>use_W</tabstop>
  <tabstop>W_pen_w</tabstop>
  <tabstop>W_pen_color</tabstop>
  <tabstop>draw_symbol</tabstop>
  <tabstop>S_option</tabstop>
  <tabstop>size</tabstop>
  <tabstop>draw_line</tabstop>
  <tabstop>L_option</tabstop>
  <tabstop>use_input</tabstop>
  <tabstop>position_input</tabstop>
  <tabstop>use_mouse</tabstop>
  <tabstop>position</tabstop>
  <tabstop>use_file</tabstop>
  <tabstop>b_datafile</tabstop>
  <tabstop>datafile</tabstop>
  <tabstop>other_options</tabstop>
  <tabstop>bok</tabstop>
  <tabstop>bexit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
